src/applications/ivc-champva/10-7959f-1/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { format, isValid } from 'date-fns';
+import { connect } from 'react-redux';
+
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import { focusElement } from 'platform/utilities/ui';
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('h2');
+    scrollToTop('topScrollElement');
+  }
+
+  render() {
+    const { form } = this.props;
+    const { submission, formId, data } = form;
+
+    const submitDate = new Date(submission?.timestamp);
+
+    const { fullName } = data;
+
+    return (
+      <div>
+        <div className="print-only">
+          <img
+            src="https://www.va.gov/img/design/logo/logo-black-and-white.png"
+            alt="VA logo"
+            width="300"
+          />
+          <h2>Application for Mock Form</h2>
+        </div>
+        <h2 className="vads-u-font-size--h3">
+          Your application has been submitted
+        </h2>
+        <p>We may contact you for more information or documents.</p>
+        <p className="screen-only">Please print this page for your records.</p>
+        <div className="inset">
+          <h3 className="vads-u-margin-top--0 vads-u-font-size--h4">
+            Foreign Medical Program (FMP) Registration Form Claim{' '}
+            <span className="vads-u-font-weight--normal">(Form {formId})</span>
+          </h3>
+          {fullName ? (
+            <span>
+              for {fullName.first} {fullName.middle} {fullName.last}
+              {fullName.suffix ? `, ${fullName.suffix}` : null}
+            </span>
+          ) : null}
+
+          {isValid(submitDate) ? (
+            <p>
+              <strong>Date submitted</strong>
+              <br />
+              <span>{format(submitDate, 'MMMM d, yyyy')}</span>
+            </p>
+          ) : null}
+          <button
```

Comment:
Is there a reason why you are not using [`<va-button>`](https://design.va.gov/storybook/?path=/docs/uswds-va-button--primary) here?

Reply:
No, I just hadn't fixed any issues that weren't causing errors yet, because I wanted to get the scaffold up first. I can fix this real quick and push though.